origin-dapp/src/css/app.css

```diff
@@ -5392,6 +5407,47 @@ a:not([href]):not([tabindex]).span-link {
   }
 }
 
+/* sm - */
```

Comment:
@sparrowDom does this mean "sm and smaller" or "smaller than sm"? I think it's the latter, in which case I would recommend `xs` as the comment.

Reply:
👍